content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -186,11 +186,7 @@ PRISMA_MIGRATION_ENGINE_BINARY=custom/my-migration-engine-unix
 PRISMA_INTROSPECTION_ENGINE_BINARY=custom/my-introspection-engine-unix
 ```
 
-<Admonition type="warning">
-
-- The Introspection Engine is served by the Migration Engine from [4.9.0](https://github.com/prisma/prisma/releases/tag/4.9.0). Therefore, the `PRISMA_INTROSPECTION_ENGINE` environment variable will not be used.
-
-</Admonition>
+### CLI Binary Targets
```

Comment:
Why move this headline up here? `PRISMA_FMT_BINARY` has nothing to do with "CLI Binary Targets".

Reply:
I must have made a mistake when applying the feedback. I'll make the change in another PR.

Reply:
Should be covered by https://github.com/prisma/docs/pull/4502